NLog.Web/LayoutRenderers/AspNetItemValueLayoutRenderer.cs

```diff
@@ -56,18 +56,14 @@ public class AspNetItemValueLayoutRenderer : LayoutRenderer
         /// </summary>
         /// <param name="builder">The <see cref="StringBuilder"/> to append the rendered data to.</param>
         /// <param name="logEvent">Logging event.</param>
-        protected override void Append(StringBuilder builder, LogEventInfo logEvent)
+        protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
```

Comment:
I'm doubting if this is backwards-compatible when we create a subclass of AspNetItemValueLayoutRenderer. There are two cases:

1 ) override the full behavior: before override `Append`.  Now we need to override DoAppend
2) Call `base.Append` - this still works, this has now also other behaviour

What do you think?